### PR TITLE
[docs] Add PyPI for plugin installation

### PR DESCRIPTION
**Pull request by grqz**
- Add syntax highlighting: shell
- Update docs: add plugin installation approach: PyPI

>[!NOTE]
> Merge after first PyPI release, at least after #10 
> __EDIT__: PyPI release created

### CODE REVIEW

**Comment by grqz** [comment]
Do we actually need yt-dlp-get-pot>=0.0.2?

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -32,6 +32,15 @@ node build/main.js
 
 ### 2. Install the plugin
 
-Make sure you have [coletdjnz's POT plugin framework](https://github.com/coletdjnz/yt-dlp-get-pot) installed already.
+Github:
+
+1. Make sure you have [coletdjnz's POT plugin framework](https://github.com/coletdjnz/yt-dlp-get-pot) installed already (must be at least version 0.0.2 or newer).
+2. Download the latest release zip from [releases](https://github.com/Brainicism/bgutil-ytdlp-pot-provider/releases). Install it by placing the zip into one of the [plugin folders](https://github.com/yt-dlp/yt-dlp#installing-plugins).
+
+PyPI:
```
[the remark this answers is not in the document]
Comment (grqz) [reply README.md:40]: did you mean moving the github instructions after the pypi one?

[the remark this answers is not in the document]
Comment (Brainicism) [reply README.md:40]: yes

[the remark this answers is not in the document]
Comment (grqz) [reply README.md:35]: done in 77df1e78b5ff80deee1dd5b46e1197eacc3f174b

**Review by Brainicism — APPROVED** [review]

**Comment by grqz** [comment]
you can merge it now if there isn't any issues

**Pull request merged**